yarn.lock

```diff
@@ -2449,9 +2458,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.30001517":
-  version: 1.0.30001520
-  resolution: "caniuse-lite@npm:1.0.30001520"
-  checksum: 59991ad8f36cf282f81abbcc6074c3097c21914cdd54bd2b3f73ac9462f57fc74e90371cd22bcdff4d085d09da42a07dcea384cb81e4ac260496e1bd79e1fe7c
+  version: 1.0.30001521
```

Comment:
Revert changes in yarn.lock, there shouldn't be changes in this file when there isn't changes in package.json

Reply:
That's not entirely true. Since we're using ^ (caret) on many of the package versions, we tell yarn to install that version or the latest minor or patch version, which is what has happened here.